src/server/modules/user/__tests__/UserController.test.ts

```diff
@@ -107,7 +107,9 @@ describe('UserController', () => {
       const res: any = httpMocks.createResponse()
       res.badRequest = jest.fn()
 
-      urlManagementService.createUrl.mockRejectedValue(new ValidationError(''))
+      urlManagementService.createUrl.mockRejectedValue(
+        new ValidationError('', []),
+      )
```

Comment:
could you share what API changes were made in sequelize that necessitated this change?

Reply:
This happens because, the ValidationError constructor is expecting 2 parameters instead of 1.
`constructor(message: string, errors: ValidationErrorItem[], options?: ErrorOptions);`